libs/api/domains/official-journal-of-iceland-application/src/lib/ojoiApplication.resolver.ts

```diff
@@ -0,0 +1,40 @@
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { Scopes } from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+import { FeatureFlag, Features } from '@island.is/nest/feature-flags'
+import { OfficialJournalOfIcelandApplicationService } from './ojoiApplication.service'
+import { GetCommentsInput } from '../models/getComments.input'
+import { GetCommentsResponse } from '../models/getComments.response'
+import { PostCommentInput } from '../models/postComment.input'
+import { PostCommentResponse } from '../models/postComment.response'
+import { PostApplicationInput } from '../models/postApplication.input'
+
+@Scopes(ApiScope.internal)
+@FeatureFlag(Features.officialJournalOfIceland)
+@Resolver()
+export class OfficialJournalOfIcelandApplicationResolver {
```

Comment:
You have added the `Scopes` decorator so you also need to add `@UseGuards(IdsUserGuard, ScopesGuard` to enable auth token validation and scope checking.